libraries/Microsoft.Bot.Builder.Expressions.Parser/ExpressionEngine.cs

```diff
@@ -16,15 +16,18 @@ namespace Microsoft.Bot.Builder.Expressions.Parser
     /// </summary>
     public class ExpressionEngine : IExpressionParser
     {
-        private static readonly Dictionary<string, string> ShorthandFunctionMap = new Dictionary<string, string>()
+        /// <summary>
+        /// constant short hand currently have.
+        /// </summary>
+        private static readonly Dictionary<string, string> ShorthandPrefixMap = new Dictionary<string, string>()
         {
-            { "#", ExpressionType.Intent },
-            { "@", ExpressionType.SimpleEntity },
-            { "@@", ExpressionType.Entity },
-            { "$", ExpressionType.Dialog },
-            { "^", ExpressionType.Callstack },
-            { "%", ExpressionType.Option },
-            { "~", ExpressionType.Instance },
+            { "#", $"turn.recognized.intents" },
+            { "@", $"turn.recognized.entities" },
+            { "@@", $"turn.recognized.entities" },
+            { "$", $"dialog" },
+            { "^", $"" },
```

Comment:
why this is not expanded?

Reply:
^ represent "Callstack" function, it has its own logical function, not just a shorthand for a accessor.